uwblueprint/headsupguys
# Auth landing page

## Notion ticket link

[\[Auth\] Implement user sign on to Landing Page](https://www.notion.so/uwblueprintexecs/Auth-Implement-user-sign-on-to-Landing-Page-88ac6787cc224ff487340c23528ce8b7)



## Implementation description
* Use `Auth.currentAuthenticatedUser()` to get authenticated user information
* Display user's name if authenticated



## Steps to test
1. Go to http://localhost:3000/
2. Observe that header says "WELCOME TO YOUR TOOLKIT"
3. Go to http://localhost:3000/login
4. Log in, or sign up and log in (note the name you use)
5. Observe redirect to http://localhost:3000/
6. Observe that header says "WELCOME TO YOUR TOOLKIT, \[NAME\]"
7. Please also take a look at [documentation](https://www.notion.so/uwblueprintexecs/Authentication-455747d7459f4e969c768837621aa5bb) (see User section)



## What should reviewers focus on?
* Does it work?
* Does implementation make sense?


## Checklist
